### PR TITLE
august 2015 meetup blog post

### PR DESCRIPTION
**Pull request by aaronj1335**
pertains to #19

@beaktor and @tgrochowicz let me know if the location details look correct.

@iheanyi let me know if the bio looks good.

@lawnsea did i forget anything?

### CODE REVIEW

File: _includes/give-em-the-business.html [hunk 1]
```diff
@@ -2,9 +2,15 @@
   <li>
     When: <x-date format="meetup">{{ page.when | date_to_xmlschema }}</x-date>
   </li>
-  <li>
-    Where: Frog Design – 101 West 6<sup>th</sup> Street – 2<sup>nd</sup> Floor
-  </li>
+  {% if include.location == "spredfast" %}
+    <li>
+      Where: Spredfast – 200 West Cesar Chavez Street – 3<sup>rd</sup> Floor of the Scilicon Labs building
```
Comment (tgrochowicz) [inline _includes/give-em-the-business.html]: s/Scilicon/Silicon

File: _includes/give-em-the-business.html [hunk 2]
```diff
@@ -31,3 +37,7 @@
   @{{ site.twitter.user }}</a>
 </p>
 
+<p>
+  And finally, <a href="/austinjs-code-of-conduct/">be excellent to eachother</a>.
```
Comment (beaktor) [inline _includes/give-em-the-business.html]: may need a space here for `eachother`

File: _posts/2015-08-18-meetup.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- posts
+- meetups
+when: 2015-08-18T19:30:00-05:00
+---
+
+<div class="ajs-box">
+  We're not in our usual location! We'll be meeting at Spredfast this month in the Silicon Labs building, so please plan accordingly.
+</div>
+
+Hey Austin Javascripters, summer is in full swing and `{% raw %}{{ joke_about_how_ember_is_hot }}{% endraw %}`. We're excited to hear from [Iheanyi Ekechukwu][] as he tells us about "Ember.js, Ember-CLI, and how it all relates to that 'shut up and dance with me' song," (which we were acutally [hoping to hear about][] last month). Hope to see you there this Tuesday, <x-date>{{ site.categories.meetups[0].when | date_to_xmlschema }}</x-date>.
```
Comment (beaktor) [inline _posts/2015-08-18-meetup.md]: `acutally` --> `actually`

**Comment by aaronj1335** [comment]
wow you guys are great, i've got some changes here at work you think you could take a quick look at those as well?

**Comment by beaktor** [comment]
@aaronj1335 sure thing, $200/hr, where do I send the invoice? :)

**Comment by iheanyi** [comment]
Looks gewd to me.

**Comment by lawnsea** [comment]
:+1: with @beaktor's fixes.

**Pull request closed without merging**